src/main/java/executor/command/CommandGetUserBalance.java

```diff
@@ -0,0 +1,32 @@
+package executor.command;
+
+import executor.task.TaskList;
+import ui.Wallet;
+import ui.Ui;
+
+public class CommandGetUserBalance extends Command {
+
+    /**
+     * Constructor for the CommandUpdateBalance class.
```

Comment:
Incorrect JavaDoc.

Reply:
Yup i have removed the class now :) I think we both did the same operation